charts/cluster-addons/values.yaml

```diff
@@ -223,7 +223,7 @@ monitoring:
             compactor:
               retention_enabled: true
             limits_config:
-              retention_period: 744h
+              retention_period: 24h
```

Comment:
I think that 24h may be a little low, especially if this is for how long logs should be kept for, as it won't give users the chance to spot any abnormalities that may occur over, let's say, a weekend.

Reply:
Our take was, things go really bad when this overflows, and most logs keep happening again and again.

However, ensuring you can check what happened Friday at 5pm on a Monday morning is a very good point. I would be OK trying for 72h instead here.